var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -48,13 +49,18 @@ class Cp2k(Package):
     depends_on('blas')
     depends_on('fftw')
     depends_on('libint@:1.2', when='@3.0')
+    depends_on('libint@:1.2', when='@4.1')
```

Comment:
I can do it, but it seems `cp2k` web-site does not provide instructions to build old versions (or at least I can't find them). I opted then to have a per-version rule so that we can keep track of restrictions on the version of some dependencies. Maybe I can turn this into:
```python
depends_on('libint@:1.2', when='@3.0,4.1')
```
What do you think?